src/helpers/bluesky/__tests__/create-media-record.test.ts

```diff
@@ -0,0 +1,45 @@
+import { createMediaRecord } from "../create-media-record";
+
```

Comment:
Would be great to rename the file into `*.spec.ts` format, so that we keep the same naming across the project